src/app/containers/Navigation/index.jsx

```diff
@@ -32,12 +40,18 @@ const NavigationContainer = () => {
   const { script, translations, navigation, service, dir } = useContext(
     ServiceContext,
   );
+  const { canonicalLink, origin } = useContext(RequestContext);
   const { currentPage, sections } = translations;
 
   if (!navigation || navigation.length === 0) {
     return null;
   }
 
+  const matchingLink = linkUrl => `${origin}${linkUrl.url}` === canonicalLink;
+  const activeIndex = navigation
+    .map(matchingLink)
```

Comment:
Do you need to map first here or could you just have the logic within findIndex?